README.md

```diff
@@ -24,30 +24,30 @@ that reads from the feed.
  */
 
 //create a secret-stack instance and add ssb-db, for persistence.
-var createSbot = require('secret-stack')({})
+var createSsbServerWithDb = require('secret-stack')({})
   .use(require('ssb-db'))
 
 
 // create the db instance.
 // Only one instance may be created at a time due to os locks on port and database files.
 
-var sbot = createSbot(require('ssb-config'))
+var ssbServerWithDb = createSsbServerWithDb(require('ssb-config'))
 
 //your public key, the default key of this instance.
 
-sbot.id
+ssbServerWithDb.id
 
 //or, called remotely
 
-sbot.whoami(function (err, data) {
+ssbServerWithDb.whoami(function (err, data) {
```

Comment:
@christianbundy do you know if `whoami` comes with `ssb-db` or something else?

Reply:
Yep, it comes from ssb-db!